Web.md

```diff
@@ -202,6 +202,95 @@ Given these values, to process a potential WebAssembly response:
       _reason_.
 1. Return _returnValue_.
 
+### Structured serialization of a `WebAssembly.Memory`
+
+A `WebAssembly.Memory` is a
+[serializable object](https://html.spec.whatwg.org/multipage/infrastructure.html#serializable-objects), and will require changes to the
+[HTML Standard](https://html.spec.whatwg.org/multipage/infrastructure.html#serialization-steps) as follows:
+
+[StructuredSerializeInternal](https://html.spec.whatwg.org/multipage/infrastructure.html#structuredserializeinternal) adds a new step after step 12:
+
+* Otherwise, if *value* has a [[WebAssembly.Memory]] internal slot, then:
+  1. Let *memory* be *value*.[[WebAssembly.Memory]], which is a
+     [`memory instance`](https://webassembly.github.io/spec/exec/runtime.html#memory-instances).
+  2. Let *data* be *memory*'s `data` field, which is a sizable vector of bytes.
```

Comment:
I'm confused. Is _data_ a Shared Data Block, or a sizable vector of bytes? Are the two concepts the same somehow?